### PR TITLE
Adding license and updating package.json

### PR DESCRIPTION
**Pull request by mkillianey**
Would you mind clarifying the preferred attribution for this module?

### CODE REVIEW

**Comment by dcousens** [comment]
Same with the others

File: package.json [hunk 1]
```diff
@@ -6,7 +6,10 @@
   "scripts": {
     "test": "node test.js | tspec"
   },
-  "author": "",
+  "author": "Calvin Metcalf <calvin.metcalf@state.ma.us>",
```
Comment (dcousens) [inline package.json]: This implies 1 author, which may change over time.  Up to @calvinmetcalf I guess

File: package.json [hunk 2]
```diff
@@ -6,7 +6,10 @@
...
+  "contributors": [
+    "Daniel Cousens <github@dcousens.com>"
+  ],
```
Comment (dcousens) [inline package.json]: Probably just don't include this at all?

**Comment by mkillianey** [comment]
Changed LICENSE to "browserify-rsa contributors" as per conversation in https://github.com/crypto-browserify/browserify-sign/pull/21

**Comment by calvinmetcalf** [comment]
lets change the line to `Calvin Metcalf & contributors` and we're good

**Comment by mkillianey** [comment]
Done

**Comment by dcousens** [comment]
Thanks @mkillianey

**Comment by calvinmetcalf** [comment]
:tada:

**Pull request merged**